### ISSUE TITLE
The ol-maker in case of multiple use, it is not clear

### ISSUE THREAD

**Issue by mfdev1** [tombatossals/angular-openlayers-directive#128]
Hi,

the ol-marker when you use more than one, you can not clear any of the markers.
By clearing all the marker item, but the pin is cleared, what can not be cleared with ol-marker unit?

Or used incorrectly?

Example:

```javascript
<!DOCTYPE html>
<html ng-app="demoapp">
<head>
    <script src="../bower_components/openlayers3/build/ol.js"></script>
    <script src="../bower_components/angular/angular.min.js"></script>
    <script src="../bower_components/angular-sanitize/angular-sanitize.min.js"></script>
    <script src="../dist/angular-openlayers-directive.js"></script>
    <link rel="stylesheet" href="../bower_components/openlayers3/build/ol.css" />
    <link rel="stylesheet" href="../dist/angular-openlayers-directive.css" />
    <script>
        var app = angular.module("demoapp", ["openlayers-directive"]);
        app.controller("DemoController", [ '$scope', function($scope) {
            var markers = [
                [
                    {
                        name: 'London',
                        lat: 51.505,
                        lon: -0.09
                    },
                    {
                        name: 'Bath',
                        lat: 51.375,
                        lon: -2.35
                    },
                    {
                        name: 'Canterbury',
                        lat: 51.267,
                        lon: 1.083
                    }
                ],
                [
                    {
                        name: 'finisterre',
                        lat: 42.907800500000000000,
                        lon: -9.265031499999964000
                    },
                    {
                        name: 'santiago',
                        lat: 42.880596200000010000,
                        lon: -8.544641200000001000
                    }
                ]
            ];
            angular.extend($scope, {
                center: {
                    lat: 47.73207540031075,
                    lon: -3.5611064500048597,
                    zoom: 5
                },
                markers: [markers[0], markers[1]]
            });

            $scope.toggleMarker = function(index) {
                if ($scope.markers[index].length > 0) {
                    $scope.markers[index] = [];
                } else {
                    $scope.markers[index] = markers[index];
                }
            };
        } ]);
    </script>
</head>
<body ng-controller="DemoController">
<openlayers ol-center="center" height="400px" width="100%">
    <ol-marker ng-repeat="marker in markers[0]" lat="marker.lat" lon="marker.lon"></ol-marker>
    <ol-marker ng-repeat="marker in markers[1]" lat="marker.lat" lon="marker.lon"></ol-marker>
</openlayers>

<h1>Multi Markers example</h1>
<p>markers index 0</p>
<input type="button" ng-click="toggleMarker(0)" value="Add/remove marker" />
<pre ng-bind="markers[0] | json"></pre>
<br>

<p>markers index 1</p>
<input type="button" ng-click="toggleMarker(1)" value="Add/remove marker" />
<pre ng-bind="markers[1] | json"></pre>
</body>

</html>
```

**Comment by tombatossals** [comment]
Hi @mfdev1, I'm sorry I don't fully understand the problem you're reporting.

I can see the markers dissapearing/appering then you click the toggle button in your example.

Can you provide a jsfiddle showing the related behaviour?

You can start from this fiddle:
http://jsfiddle.net/tombatossals/v3b0p54h/

**Comment by mfdev1** [comment]
Hi @tombatossals,
         
I am sorry.
Was confirmed in the latest version , the problem could not be found .

Perhaps you think that has been resolved in #110 .

I did apologize for trouble .  

Also thank you .

https://jsfiddle.net/mfdev1/wk07rroo/2/

**Comment by tombatossals** [comment]
Ah ok, don't worry, great it was solved previously.

Thanks!
